packages/rpc/src/index.ts

```diff
@@ -64,41 +64,39 @@ export const endpoints = {
   signPsbt,
   signMessage,
   sendTransfer,
-};
+} as const;
 
 type EndpointMap = (typeof endpoints)[keyof typeof endpoints];
 
-export type LeatherRpcMethodMap = {
+export type RpcEndpointMap = {
   [E in EndpointMap as E['method']]: {
     request: z.infer<E['request']>;
     response: z.infer<E['response']>;
   };
 };
 
-export type RpcRequests = ValueOf<LeatherRpcMethodMap>['request'];
+/** @deprecated */
```

Comment:
Is this a temporary deprecation? 

Will you refactor the extension to remove `LeatherRpcMethodMap`?